features/filesystem/sd/SDBlockDevice.h

```diff
@@ -127,7 +127,7 @@ class SDBlockDevice : public BlockDevice {
      */
     virtual void debug(bool dbg);
 
-private:
+protected:
```

Comment:
The comment in the file says

```
 * Firstly, a low initial clock should be selected (in the range of 100-
 * 400kHZ). After initialisation has been completed, the switch to a
 * higher clock speed can be made (e.g. 1MHz). Newer cards will support
 * higher speeds than the default _transfer_sck defined here.
```

But can't find any API to change the clocks (I believe you are after ``_transfer_sck`` to be updated).

@geky - would it make sense to add frequency() method, as its for SPI class